### PR TITLE
Relay-term: Removed meta-nodejs, use yocto's native nodejs, removed dependency on global-node-modules

### PR DESCRIPTION
**Pull request by ygoyal18**
Also added shrinkwrap.json as its required to successfully build the package. npm-shrinkwrap.json is also part of the relay-term source code, but was not able to use that so had to add it here.

### CODE REVIEW

**Comment by ygoyal18** [comment]
Have kicked off a build to manually test this.

**Review by x85446 — APPROVED** [review]

**Review by cmonr — DISMISSED** [review]
LGTM, assuming the build is successful

**Review by x85446 — DISMISSED** [review]

File: recipes-wigwag/relay-term/relay-term_0.0.1.bb [hunk 1]
```diff
@@ -26,16 +32,19 @@ ${PN}-watcher.path"
...
     install -m 755 ${WORKDIR}/${PN}-watcher.service ${D}${systemd_system_unitdir}/${PN}-watcher.service
     install -m 755 ${WORKDIR}/${PN}-watcher.path ${D}${systemd_system_unitdir}/${PN}-watcher.path
-}
-
 
+    install -d ${D}/wigwag/devicejs-core-modules/node_modules
+    cp -r ${S}/node_modules ${D}/wigwag/devicejs-core-modules/node_modules
 
+    install -d ${D}/wigwag/wigwag-core-modules
+    cp -r ${S} ${D}/wigwag/wigwag-core-modules/relay-term
```
Comment (costanic) [inline recipes-wigwag/relay-term/relay-term_0.0.1.bb:49]: What is ${S} pointing to?  Does this copy all of edge-node-modules into the relay-term folder, for example DevStateManater, LEDController, RelayStatsSender, etc.?

File: recipes-wigwag/relay-term/relay-term_0.0.1.bb [hunk 2]
```diff
@@ -5,19 +5,25 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
 
 RT_SERVICE_FILE = "pelion-relay-term.service"
 PR = "r1"
+S = "${WORKDIR}/git/relay-term"
```
Comment (cmonr) [inline recipes-wigwag/relay-term/relay-term_0.0.1.bb:8]: @costanic 

Not sure why the GitHub view didn't show this as well...

**Comment by ygoyal18** [comment]
Manually verified it on a dev build.

**Review by x85446 — APPROVED** [review]

**Pull request merged**